go.mod

```diff
@@ -1,6 +1,7 @@
 module kubevirt.io/containerdisks
 
 go 1.21
+toolchain go1.22.2
```

Comment:
I think this is okay and likely means we can bump the `go` line above to 1.22 now?